### PR TITLE
fix: Optimus will now set identity level

### PR DESCRIPTION
**Pull request by 3Hren**

### CODE REVIEW

File: etc/optimus.yaml [hunk 1]
```diff
@@ -49,6 +49,12 @@ workers:
     # Optional.
     # Default value is false.
     # dry_run: true
+    # Identity level specifies which kind of customers this bot allows to work
+    # with.
+    # Required.
+    # Possible values are: unknown, anonymous, registered, identified and
```
Comment (antmat) [inline etc/optimus.yaml]: I don't think we should mention `unknown` in comments. Moreover - check for non-zero value is a good idea

Comment (antmat) [reply etc/optimus.yaml]: Ah, it's already required, so should not be zero. Then remove `unknown` from comment please

File: etc/optimus.yaml [hunk 2]
```diff
@@ -49,6 +49,12 @@ workers:
     # Optional.
     # Default value is false.
     # dry_run: true
+    # Identity level specifies which kind of customers this bot allows to work
+    # with.
+    # Required.
+    # Possible values are: anonymous, registered, identified and
+    # professional.
```
Comment (antmat) [inline etc/optimus.yaml]: just curious - what's the purpose of newline here?

Comment (3Hren) [reply etc/optimus.yaml]: 80 margin limit

**Pull request merged**